src/Container/ApplicationFactory.php

```diff
@@ -13,6 +13,7 @@
 use Zend\Diactoros\Response\EmitterInterface;
 use Zend\Expressive\Application;
 use Zend\Expressive\Exception;
+use Zend\Expressive\Container\Exception\InvalidArgumentException as CntInvalidArgumentException;
```

Comment:
I prefer `ContainerInvalidArgumentException`. I didn't know what Cnt means at the begin.